tests/conftest.py

```diff
@@ -166,75 +200,143 @@ def fixture_t4c_exporter_env(
 
 
 @pytest.fixture(name="init_t4c_server_repo")
-def fixture_init_t4c_server_repo(t4c_http_port: str):
-    create_t4c_repository(t4c_http_port)
+def fixture_init_t4c_server_repo(t4c_ip_addr: str, t4c_http_port: str):
+    create_t4c_repository(t4c_ip_addr, t4c_http_port)
     yield
 
 
 @contextlib.contextmanager
 def get_container(
     image: str,
+    image_tag_env: str | None = None,
```

Comment:
IMO we should keep this approach as by doing so we only need to have the checks we do here once. Otherwise, we would have to check whether the environment variable exists in each fixture where we potentially have an env variable set leading to code duplication.